### DISCUSSION TITLE
I cannot set my fallback SMS on my acount (under Password & Authentication)

### DISCUSSION THREAD

**Discussion by jameydev** [community/community#38355] [Other Feature Feedback, Questions, & Ideas] [answered]
I can't seem to figure out exactly what happened. I have bad cellular signal so I had to try to get it to send the SMS code several times, I finally got it and I accidentally clicked out of the dialog to verify the code and when I tried again nothing happened. According to the chrome developer console, two things are happening here:
1) An HTTP response of 400 - bad request to the API
2) An HTTP response of 422 to https://github.com/settings/two_factor_authentication/setup/backup_number which means that the request was syntactically correct, but somehow semantically not?
Basically, in the actually UI, I click set fallback after entering my mobile number and nothing happens.
Somehow the request got messed up and reloading the page doesn't work. Not even a hard reload and emptying the cache works.
I guess somehow it thinks I made too many requests, but keep in mind I was having trouble receiving the SMS code.
What do I do?
Screenshot of page I'm talking about (without my number for security/privacy):
![image](https://user-images.githubusercontent.com/25995607/200123160-ab67ade0-ef74-4250-8541-85bf762edd55.png)

Screenshot from developer console with the actual POST errors:
![image](https://user-images.githubusercontent.com/25995607/200123184-702818d1-3a7e-4b51-9466-af2c99084b05.png)

This is really frustrating and waiting a few minutes doesn't seem to help either. I am just trying to keep my account secure like what the heck.

**Comment by jameydev** [comment, marked as answer]
Hey all, I apparently just had to wait like over an hour before sending the code again. I hope this helps anyone who has a similar issue. My fallback number for my account is now set.

**Reply by Gainville12** [reply]
Sms

**Reply by Gainville12** [reply]
J'arrive pas à me envoyé des messages il veuillez installer l'application bt notifier
